dotnet/src/webdriver/DevTools/DevToolsSession.cs

```diff
@@ -55,6 +55,8 @@ public class DevToolsSession : IDevToolsSession
         private CancellationTokenSource receiveCancellationToken;
         private Task receiveTask;
 
+        private SemaphoreSlim semaphoreSlimForSocketSend = new SemaphoreSlim(1);
```

Comment:
I believe that `maxCount` constructor parameter of `SemaphoreSlim` should also be set to `1`. Additionally, the field can be read-only. 

```suggestion
        private readonly SemaphoreSlim semaphoreSlimForSocketSend = new SemaphoreSlim(1, 1);
```